### PR TITLE
PRM: add less hostile wildlife: I of II

### PR DESCRIPTION
**Pull request by pisskop**
Adds a large dog, a goat, and a skunk

### CODE REVIEW

**Comment by mugling** [comment]
Presuming these load without error I can't see a problem when compared to previous batch

File: data/mods/PKs_Rebalance/pk_critter_misc.json [hunk 1]
```diff
@@ -456,37 +492,73 @@
...
+    "dodge":2,
+    "vision_day":35,
     "vision_night":3,
-    "hp":450,
+    "hp":12,
     "death_function":"NORMAL",
-    "special_attacks":[["GRAB", 30], ["VINE", 15]],
-    "description":"One of the most durable creatures in the world, the tardigrade has outlived the dinosaurs and humanity both.  This is a plump and now moose-sized eight-legged critter with a creviced shell with in-laid segments.  Along the length of the shell pairs of sensory tentacles can be seen waving around.  Although its clawed digits seem to be for grasping the tentacles, due to their sheer size, lend themselves to flailing in defense.",
-    "flags":["SEES", "SMELLS", "PATH_AVOID_DANGER_1", "STUMBLES", "BASHES", "ARTHROPOD_BLOOD", "NO_BREATHE", "SWIMS", "ANIMAL", "FAT","CHITIN"],
+    "description":"A small mammal with white stipped black fur that feeds on insects and fruit, roots, and seeds.   Known to spray a foul smelling liquid if distrubed.",
```
Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_critter_misc.json]: typo in the end of the sentence: ```distrubed```.

**Comment by pisskop** [comment]
are there errors

**Comment by mugling** [comment]
@pisskop can you check the output of `make check-lint` works for you

File: data/mods/PKs_Rebalance/pk_critter_misc.json (hunk 1 above)
Comment (Snaaty) [inline data/mods/PKs_Rebalance/pk_critter_misc.json]: Another typo: "A small mammal with white stipped" -> "striped"

**Comment by pisskop** [comment]
It claims there is no such rule

**Comment by mugling** [comment]
> Perhaps now is a time to claim that I do not maintain a local head for the remote repository?

It was already obvious this was the case. It would explain the recurrent JSON errors and bizarrely chunked PR's. 

> It claims there is no such rule.

Sorry, typo on my part try `make lint-check`. Try that again

**Comment by pisskop** [comment]
Why are you asking me to check a lint for items when this pr does not update any item files.

Dont jank me around

make lint-check
make: *** No rule to make target 'data/mods/PKs_Rebalance/items/*.json', needed by 'obj/lint.cache'.  Stop.

**Comment by mugling** [comment]
> Why are you asking me to check a lint for items when this pr does not update any item files.

Because the diagnostic message is really helpful - you have a reasonably recent copy of `master`

**Comment by mugling** [comment]
> Why are you asking me to check a lint for items when this pr does not update any item files.

Because the diagnostic message is helpful - it shows you have a reasonably recent copy of `master` (that includes the new `Makefile` target) but the contents of your `mods` dir is entirely different.

> are there errors
> Dont jank me around

By definition if you aren't tracking `master` you aren't checking your PR's. Requiring other developers to pull a branch to find it contains JSON load errors is antisocial. It also leads to frequent breakage of your mod and time is wasted reviewing changes that don't even work (many examples #16193, #15915 etc).

Secondly you have a competing version you publish on the forum. There are now innumerable variants of your mod in circulation, the one in `master`, the one you publish and then the one formed by combining any random copy of `master`

**Comment by mugling** [comment]
> Why are you asking me to check a lint for items when this pr does not update any item files.

Because the diagnostic message is helpful - it shows you have a reasonably recent copy of `master` (that includes the new `Makefile` target) but the contents of your `mods` dir is entirely different.

> are there errors
> Dont jank me around

By definition if you aren't tracking master you aren't checking your PR's. As a result other developers pull a branch to then find it contains JSON load error. It also leads to frequent breakage of your mod and time is wasted reviewing changes that don't even work (many examples #16193, #15915 etc).

Secondly you have a competing version you publish on the forum. There are now innumerable variants of your mod in circulation: the one in master, the one you publish and then the one formed by combining any random copy of `master` with a downloaded version. Point in case is the bug report in #18277 - is it broken because you didn't test, is it mutually incompatible versions or is it a bug in `core`?

You need to apply your changes to `master` as copy-and-paste has lead to the bad syntax as well as unwieldy PR's that include irrelevant content. The last time I did any automated checking I found a lot of errors that probably occurred due to this exact process.

I'm going to merge test this and then lintify your monster definitions. Before you submit anything further please update to `master`.

Testing your PR's is neither a new rule nor specific to you. Before **every** commit please run:
```
make lint
./cataclysm-tiles --check-mods PK_REBALANCE
```

**Pull request merged**